students/itsabhi/Lesson6/test_mailroom.py

```diff
@@ -0,0 +1,45 @@
+import mailroom4v1
+import unittest
+from unittest.mock import patch
+import os
+
+
+class DonationsTestCases(unittest.TestCase):
+
+    def setUp(self):
+    #Invoked before each test method
+        self.test_donors_dict = {
+        'A G': [10, 20, 30],
+        'N D': [40, 50]
+    }
+    def test_list_donors(self):
+    #Test donor list when the option "list" is selected under the Sending Thanks menu
```

Comment:
There are a few Pep8 violations in this file, I would recommend using a code linter